config/settings.yml

```diff
@@ -1,3 +1,4 @@
 # Used to add feature flags in the app to control access to certain features.
 features:
   make_question_optional: true
```

Comment:
We should have a ticket to remove this feature flag  and any conditions

Reply:
Have made one for that here: https://trello.com/c/RsRgO2kq/187-remove-optional-questions-feature-flag